explainer.md

```diff
@@ -1,624 +1,397 @@
-[here](https://material.io/design/motion/the-motion-system.html#transition-patterns).
-The following is a summary of the semantics of these transition patterns :
-
-*   Root Transitions : The full page content animates between two web pages with
-    an optional static UI element on top.
-    [Shared axis](https://material.io/design/motion/the-motion-system.html#shared-axis)
-    shows an example.
-*   Shared Element to Root Transitions : A persistent UI element morphs into the full
-    page content on the next web page.
-    [Container transform](https://material.io/design/motion/the-motion-system.html#container-transform)
-    shows an example.
-*   Shared Element Transitions : A persistent UI element morphs into another
-    UI element on the next web page. The element's contents and shape can change during
-    this transition. This [video](https://www.youtube.com/watch?v=SGnZN3NE0jA) shows an
-    example.
-*   Entry/Exit Transitions : A UI element animates as it exits or enters the
-    screen. This
-    [issue](https://github.com/WICG/shared-element-transitions/issues/37) shows
-    an example.
-
-These transitions should be feasible in SPAs (Single Page Apps) and MPAs (Multi
-Page Apps).
+A visual demo of some example transition patterns targeted by this feature are [here](https://material.io/design/motion/the-motion-system.html#transition-patterns). The following is a summary of the semantics of these transition patterns:
+
+* Root Transitions: The full page content animates between two web pages with an optional static UI element on top. Examples 1 & 2 [here](https://material.io/design/motion/the-motion-system.html#shared-axis) are demonstrations of this.
+* Shared Element to Root Transitions: A persistent UI element morphs into the full page content on the next web page. [Container transform](https://material.io/design/motion/the-motion-system.html#container-transform) shows an example.
+* Shared Element Transitions: A persistent UI element morphs into another UI element on the next web page. The element's contents and shape can change during this transition. This [video](https://www.youtube.com/watch?v=SGnZN3NE0jA) shows an example.
+* Entry/Exit Transitions: A UI element animates as it exits or enters the screen. This [issue](https://github.com/WICG/shared-element-transitions/issues/37) shows an example.
 
 # Design
 
-Let's take the example below which shows how the API can be used by a developer
-to animate the background and a shared element on a same origin navigation
-(MPA). The SPA equivalent of this case is one where the old document is mutated
-into the new document via DOM APIs. See [API Extensions](#api-extensions) for
-more code examples.
+The goal is to provide a mechanism and API which will allow simple transitions like above to be specified in CSS, building on CSS animations, but also allow for more complex transition to be performed via JavaScript, building on the Web Animations API.
```

Comment:
What do you mean by "animations sticking to the finger"?

Reply:
By sticking to the finger, I mean animations that track touch input. So I can slide something 30% of the way and hold my finger there, then slide it another 10% and the animation would advance another 10%. As opposed to what we're doing here in that once the animation started, it will run and finish without the user being able to affect it.

Reply:
"Is it a non-goal to allow simple transitions easily (ie with no custom CSS)?" This seems to imply that defaults are a non-goal. I don't think that's right.